contracts/interfaces/IFractalUsul.sol

```diff
@@ -2,44 +2,166 @@
+    /// @param prevStrategy Strategy that pointed to the strategy to be removed in the linked list
+    /// @param strategy Strategy to be removed
+    function disableStrategy(address prevStrategy, address strategy) external;
+
+    /// @notice This method submits a proposal which includes metadata strings to describe the proposal
+    /// @param strategy Address of Voting Strategy, under which proposal submitted
+    /// @param data - any additional data, which would be passed into IStrategy.receiveProposal
+    /// @param transactions - array of transactions to execute
+    /// @param title - proposal title, emitted in ProposalCreated
+    /// @param description - proposal description, emitted in ProposalCreated
+    /// @param documentationUrl - proposal documentation/discussion URL, emitted in ProposalCreated
+    function submitProposal(
+        address strategy,
+        bytes memory data,
+        FractalUsul.Transaction[] calldata transactions,
+        string calldata title,
+        string calldata description,
+        string calldata documentationUrl
+    ) external;
+
+    /// @notice Cancels an array of proposals
+    /// @param proposalIds Array of proposals to cancel
+    function cancelProposals(uint256[] memory proposalIds) external;
+
+    /// @notice Called by the strategy contract when the proposal vote has succeeded
+    /// @param proposalId the identifier of the proposal
+    /// @param timeLockPeriod the optional delay time
+    function timelockProposal(uint256 proposalId, uint256 timeLockPeriod) external;
+
+    /// @notice Executes the specified transaction within a proposal
+    /// @notice Transactions must be called in order
+    /// @param proposalId the identifier of the proposal
+    /// @param target the contract to be called by the avatar
+    /// @param value ether value to pass with the call
+    /// @param data the data to be executed from the call
+    /// @param operation Call or Delegatecall
+    function executeProposalByIndex(
+        uint256 proposalId,
+        address target,
+        uint256 value,
+        bytes memory data,
+        Enum.Operation operation
+    ) external;
+
+    /// @notice Executes all the transactions within a proposal
+    /// @param proposalId the identifier of the proposal
+    /// @param targets the contracts to be called by the avatar
+    /// @param values ether values to pass with the calls
+    /// @param data the data to be executed from the calls
+    /// @param operations Calls or Delegatecalls
+    function executeProposalBatch(
+        uint256 proposalId,
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory data,
+        Enum.Operation[] memory operations
+    ) external;
+
+    /// @notice Returns if a strategy is enabled
+    /// @param _strategy The address of the strategy to check
+    /// @return True if the strategy is enabled
+    function isStrategyEnabled(address _strategy) external view returns (bool);
```

Comment:
we should be consistent in whether we preface parameters with an underscore, I don't have a preference either way, but it seems like this is the only one with it

Reply:
Yeah, agree we should be consistent here. I do prefer having the underscore on function args, since a lot of times they would otherwise collide with global state variable names. I can change them to all have underscores